snuba/datasets/storage.py

```diff
@@ -89,6 +89,10 @@ def get_mandatory_condition_checkers(self) -> Sequence[ConditionChecker]:
         """
         return []
 
+    @abstractmethod
```

Comment:
Every storage should have a storage key. This was causing mypy issues and it is stupid